apidoc/Facebook.yml

```diff
@@ -1070,8 +1070,12 @@ properties:
     summary: |
         Opens login window with the native Facebook app. On iOS it will attempt to
```

Comment:
What do you mean? This constant only describes the `LOGIN_BEHAVIOR_NATIVE` constant which can fallback to `LOGIN_BEHAVIOR_BROWSER` internally - as described below.